src/transport/PairingSession.h

```diff
@@ -104,7 +104,7 @@ class DLL_EXPORT PairingSession
                                           TLV::TLVWriter & tlvWriter);
 
 protected:
-    void SetSecureSessionType(Transport::SecureSession::Type secureSessionType) { mSecureSessionType = secureSessionType; }
+    // void SetSecureSessionType(Transport::SecureSession::Type secureSessionType) { mSecureSessionType = secureSessionType; }
```

Comment:
could we delete commented-out code instead of keeping it?

it seems we can then also remove the "protected:" line and have a smaller file vertically.

Reply:
removed this line